.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,48 @@
+# Unique name for this workflow
+name: CI (PR)
+
+# Definition when the workflow should run
+on:
+    pull_request:
+        types: [opened, synchronize, reopened]
```

Comment:
Please add `edited`.

Reply:
Good point, we need to fix that for all samples apps.
The orignal CI flow was missing it and I noticed that some workflows from other apps are missing that part too.